spring-cloud-spanner-spring-data-r2dbc/README.md

```diff
@@ -0,0 +1,47 @@
+# Cloud Spanner Spring Data R2DBC
+
+The Spring Data R2DBC Dialect for Cloud Spanner enables the usage of [Spring Data R2DBC](https://github.com/spring-projects/spring-data-r2dbc) with Cloud Spanner.
+
+The goal of the Spring Data project is to create easy and consistent ways of using data access technologies from Spring Framework applications.
+
+## Setup
+
+To begin using Spring Data R2DBC with Cloud Spanner, add the following dependencies to your project:
+
+```xml
+<dependencies>
+    <!-- The starter dependency for the Spring Data R2DBC framework. -->
+    <dependency>
+        <groupId>org.springframework.boot.experimental</groupId>
+        <artifactId>spring-boot-starter-data-r2dbc</artifactId>
+        <version>${r2dbc-version}</version>
```

Comment:
READMEs tend to have hardcoded versions that are updated as part of the release.

Reply:
I have updated the readme and have moved it to `spring-cloud-gcp-docs/md/spanner-spring-data-r2dbc.md` . `spring-cloud-spanner-spring-data-r2dbc` version can be retrieved from the `Spring Framework on Google Cloud BOM`
@zhumin8 - Can we specify version for `spring-boot-starter-data-r2dbc`?